ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -571,4 +571,19 @@ FAN_SS_AL:
   - control_mode
   - run_mode
   - schedule_run_command
-  - control_status
\ No newline at end of file
+  - control_status
+
+FAN_SS_SC_FA_SR:
+  description: "Basic Exhaust fan with start/stop, speed sensor, fail alarm, scheduled run"
+  uses:
+  - exhaust_fan_speed_percentage_sensor
+  implements:
+  - FAN_SS_AL
+  - SRC
+
+FAN_ST:
+  description: "Basic fan with flowrate status only"
+  uses:
+  - supply_air_flowrate_status
```

Comment:
If this is directly into a space, it should be `discharge_air_flowrate_status`. It would only be `supply` if it goes to another piece of equipment.

Reply:
noted sir.